presentation/src/main/java/com/stop/ui/map/MapFragment.kt

```diff
@@ -212,81 +153,155 @@ class MapFragment : Fragment() {
-
-                        tMapView.setCenterPoint(currentTMapPoint.latitude, currentTMapPoint.longitude)
+                        val currentTMapPoint =
+                            TMapPoint(currentLocation.latitude, currentLocation.longitude)
 
-                        setPanel(currentTMapPoint)
-
-                        makeMarker(
-                            MARKER,
-                            R.drawable.ic_baseline_location_on_32,
-                            currentTMapPoint
+                        tMapView.setCenterPoint(
+                            currentTMapPoint.latitude,
+                            currentTMapPoint.longitude
                         )
+                        addPanel(currentTMapPoint)
+                        addTMapMarker(POINT_MARKER, R.drawable.ic_point_marker, currentTMapPoint)
                     }
             }
         }
     }
 
+    private fun addPanel(tMapPoint: TMapPoint) {
+        placeSearchViewModel.getGeoLocationInfo(tMapPoint.latitude, tMapPoint.longitude)
+    }
+
+    private fun setMapUiVisibility() {
+        val viewVisibility = if (isMapUiVisible) {
+            View.VISIBLE
+        } else {
+            View.GONE
+        }
+
+        with(binding) {
+            layoutSearch.visibility = viewVisibility
+            layoutCompass.visibility = viewVisibility
+            layoutCurrent.visibility = viewVisibility
+            layoutBookmark.visibility = viewVisibility
+        }
+    }
+
+    private fun convertDpToPx(dp: Int): Int {
+        return TypedValue.applyDimension(
+            TypedValue.COMPLEX_UNIT_DIP,
+            dp.toFloat(),
+            resources.displayMetrics
+        ).toInt()
+    }
+
+    private val requestPermissionsLauncher = registerForActivityResult(
+        ActivityResultContracts.RequestMultiplePermissions()
+    ) { permissions ->
+        if (permissions.entries.any { it.value }) {
+            setTrackingMode()
+        }
+    }
+
+    private val onLocationChangeListener = TMapGpsManager.OnLocationChangedListener { location ->
+        if (location != null) {
+            if (isTracking) {
+                tMapView.setCenterPoint(location.latitude, location.longitude, true)
+            }
+
+            placeSearchViewModel.currentLocation = Location(location.latitude, location.longitude)
+            addTMapMarker(
+                PERSON_MARKER,
+                R.drawable.ic_person_marker,
+                TMapPoint(location.latitude, location.longitude)
+            )
+        }
+    }
+
+    private val onEnableScrollWithZoomLevelListener =
+        TMapView.OnEnableScrollWithZoomLevelCallback { _, tMapPoint ->
+            isTracking = false
+            enablePoint.add(Location(tMapPoint.latitude, tMapPoint.longitude))
+        }
+
+    private val onDisableScrollWithZoomLevelListener =
+        TMapView.OnDisableScrollWithZoomLevelCallback { _, _ ->
+            if (enablePoint.size == SIZE_ONE) {
```

Comment:
```suggestion
            if (enablePoint.size <= SIZE_ONE) {
```

enablePoint.size가 0인 경우도 포함해야 합니다. 에뮬레이터에서는 잘 안 되고, 실제 기기에서는 잘 작동했던 이유는
에뮬레이터에서 마우스 클릭할 경우는 스크롤이 아니라 정말 클릭한 효과만 전해지기 때문에 enablePoint.size가 0이 됩니다.
반면에 실 기기에서 사용자가 손가락으로 눌렀을 때는 정말 클릭한 효과만 주기 힘들기 때문에 enablePoint.size가 0이 되기 힘듭니다.

`enablePoint.size <= SIZE_ONE`로 코드를 변경했을 때, 왜 패널이 나오자마자 바로 사라질까요?
`OnDisableScrollWithZoomLevelCallback` 과 `OnLongClickListenerCallBack` 이 동시에 실행되기 때문입니다.
`OnLongClickListenerCallBack`가 먼저 완료되서 패널이 나온 후, `OnDisableScrollWithZoomLevelCallback`에서 패널을 숨기는 부분이 동작합니다.
따라서, `OnDisableScrollWithZoomLevelCallback`에 LongClickListener가 동작했는지 확인하는 조건문을 추가해줘야 할 것 같아요.